fn-extra/src/Web/Fn/Extra/Heist.hs

```diff
@@ -89,11 +93,16 @@ heistInit :: HeistContext ctxt =>
              IO (Either [String] (FnHeistState ctxt))
 heistInit templateLocations isplices csplices =
   do let ts = map (loadTemplates . T.unpack) templateLocations
-     runEitherT $ initHeist (emptyHeistConfig & hcTemplateLocations .~ ts
-                                    & hcInterpretedSplices .~ isplices
-                                    & hcLoadTimeSplices .~ defaultLoadTimeSplices
-                                    & hcCompiledSplices .~ csplices
-                                    & hcNamespace .~ "")
+     let config = emptyHeistConfig & hcTemplateLocations .~ ts
+                                   & hcInterpretedSplices .~ isplices
+                                   & hcLoadTimeSplices .~ defaultLoadTimeSplices
+                                   & hcCompiledSplices .~ csplices
+                                   & hcNamespace .~ ""
+     #if MIN_VERSION_heist(1,0,0)
+     initHeist config
+     #else
```

Comment:
Bless you, @huggablemonad! It works fine on my machine so I was puzzled why it was failing in CI! Would you be willing to pull the branch and add your changes?